mountpoint-s3-fs/src/fs/handles.rs

```diff
@@ -39,77 +40,84 @@ where
     Client: ObjectClient + Clone + Send + Sync + 'static,
 {
     /// The file handle has been assigned as a read handle
-    Read(PrefetchGetObject<Client>),
+    Read {
+        request: PrefetchGetObject<Client>,
+        flushed: bool,
```

Comment:
"flushed" seems to be recorded in multiple places: here and in the superblock (`HandleState::Inactive`), I remember that duplication optimises something, could you remind me which specific scenario please?